mkosi/__init__.py

```diff
@@ -2084,7 +2091,7 @@ def install_kernel(context: Context, partitions: Sequence[Partition]) -> None:
     # single-file images have the benefit that they can be signed like normal EFI binaries, and can
     # encode everything necessary to boot a specific root device, including the root hash.
 
-    if context.config.output_format in (OutputFormat.uki, OutputFormat.esp):
+    if context.config.output_format in (OutputFormat.uki, OutputFormat.esp, OutputFormat.addon):
```

Comment:
Ditto

Reply:
Why? It's a different format from uki/esp

Reply:
It's handled by the below block:

```
    if context.config.bootable == ConfigFeature.auto and (
        context.config.output_format == OutputFormat.cpio
        or context.config.output_format.is_extension_or_portable_image()
        or context.config.overlay
    ):
        return
```

(And you set `Bootable=no` anyway so it'll be skipped even earlier.